docs/content/get-started/developers.mdx

```diff
@@ -5,18 +5,188 @@ nav_order: 20
+
+### Autoprefixing
+
+The design system requires autoprefixing to work properly. Don't add vendor prefixes to your custom styles manually — it is more reliable to use autoprefixing.
+
+We use the following autoprefixer settings via .browserslistrc config:
+
+```js
+> 2%
+last 2 versions
+IE 11
+not dead
+```
+
+#### About Sass Load Paths
+
+`ncids-css` requires [Sass Load Paths](https://sass-lang.com/documentation/at-rules/use#load-paths) to compile for use in your project.
+
+Load paths must include a path to the `/packages` directory for NCIDS packages and `/uswds-packages` for USWDS packages. An example using Webpack might look like:
+
+```js
+{
+		loader: 'sass-loader',
+		options: {
+			sassOptions: {
+				includePaths: [
+					path.join(
+						__dirname,
+						'./node_modules/@nciocpl/ncids-css/packages'
+					),
+					path.join(
+						__dirname,
+						'./node_modules/@nciocpl/ncids-css/uswds-packages'
+					),
+				],
+			},
+		},
+},
+```
+
+### Import NCIDS styles to your project
+
+1. Create a sass stylesheet in your project.
+
+Import the required sass partials:
+
+```sass
+@use 'uswds-core' with (
+	$theme-image-path: '@nciocpl/ncids-css/uswds-img',
+);
+@forward 'uswds-global';
+```
+
+Next, add the entire NCIDS package library or just the packages you need:
+
+- Option 1: Import the entire ncids-css library.
+
+  ```sass
+  @forward 'ncids';
+  ```
+
+- Option 2: Import only the packages you need.
+
+  ```sass
+  @forward 'nci-header';
+  @forward 'usa-breadcrumbs';
+  @forward 'usa-footer';
+  ```
+
+2. Create an index.js file. This is the entrypoint file set in the compiler of your project.
+
+```js
+import './styles.scss';
+
+console.log('hello world');
+```
+
+#### About `@use` and `@forward`
+
+The @use and @forward directives are used to import and manage SCSS modules and partials.
+
+- `@use` directive allows you to import and use functionality from other SCSS modules.
+  It provides better encapsulation and avoids conflicting global CSS class names.
+  Use it to import SCSS modules that contain reusable styles, mixins, variables, or functions.
+
+- `@forward` directive allows you to re-export styles from one module to another.
+  It acts as a bridge between different SCSS modules, making styles available for import in other files.
+  Use it when you want to expose certain styles or functionality from one module to be used in another.
+
+**Note:** The load paths option for importing Sass partials or modules is specific to **Dart Sass**. If you're using another Sass implementation, such as Node Sass, LibSass, or Ruby Sass, the load paths option may not be available. Make sure to use the appropriate load path mechanism specific to your chosen Sass implementation.
+
+### Examples
+
+For more detailed instructions on how to setup your project, visit our [example app repository](https://github.com/NCIOCPL/ncids-example-sites).
+
+## Phase 3: Customize
+
+Customize the design system with settings and custom code.
```

Comment:
We should add something here about utilities, classes, mixins etc... options.. 

USWDS goes into a lot of detail here https://designsystem.digital.gov/documentation/getting-started/developers/phase-three-customize/